Modules/Options/TrackerTab/QuestieOptionsTracker.lua

```diff
@@ -133,28 +142,34 @@ function QuestieOptions.tabs.tracker:Initialize()
                 order = 1.7,
                 width = 1.0,
                 name = function() return l10n('Enable Header'); end,
-                desc = function() return l10n('When this is checked, the Questie Tracker Header will become visible and the total number of quests you have in your log will be shown.'); end,
+                desc = function() return l10n('When this is checked, the Active Quests Header will become visible and the total number of quests you have in your log will be shown.'); end,
```

Comment:
The [translation](https://github.com/Questie/Questie/tree/master/Localization/Translations) needs to be fixed